### ISSUE TITLE
Support include-path and system-include-path

### ISSUE THREAD

**Issue by jorissteyn** [stevenremot/ede-php-autoload#9]
Despite earlier conversations about whether or not to support require/include directives, I kinda changed my mind on the subject. Altough not that useful on real-life projects, it just makes sense to implement this use-case, because it is simpler than the alternatives and gives us a good "base-line".

Anyway, what I'm missing in ede-php-autoload is the include-path and system-include-path properties. I currently work around it by using ede-cpp-project-root:

```lisp
(ede-cpp-root-project "ZF1"
                      :file "/home/joris/repos/zf1/composer.json"
                      :include-path '("/home/joris/repos/zf1/library")
                      :system-include-path '("/usr/share/php"))
```

But in order to get the best of both, I'd like to use:

```lisp
(ede-php-autoload-project "ZF1"
                           :file "/home/joris/repos/zf1/composer.json"
                           :include-path '("/home/joris/repos/zf1/library")
                           :system-include-path '("/usr/share/php"))
```
Is that a matter is changing the base class, or would it involve more work?

**Comment by stevenremot** [comment]
Hey,

sorry for the late reply, I was in vacation for the last two weeks.

Implementing this naively would only require adding class slots, so it's not a big deal. However this highlights a use case I already thought about but could not solve: using using another project type alongside ede-php-autoload. It is not possible to use two EDE projects in the same directory. For now ede-php-autoload is the only EDE project useful for PHP (I think) so t's okay, but I should keep that in mind for later.

For now if you are okay with it I can just add the slots and everything will be fine.

**Comment by stevenremot** [comment]
Hi @jorissteyn ,

I added the slots to the `ede-php-autoload-project` class. I implemented it in the branch [#9-include-path-slots](https://github.com/stevenremot/ede-php-autoload/compare/%239-include-path-slots). If you are OK with the code I will merge it to master.

**Comment by stevenremot** [comment]
Since the issue has not been updated since 3 months, I have merged the branch in master. Don't hesitate to reopen it if there is any problem.
